python/hail/methods/family_methods.py

```diff
@@ -225,4 +216,207 @@ def mendel_errors(dataset, pedigree):
 
     kts = require_biallelic(dataset, 'mendel_errors')._jvds.mendelErrors(pedigree._jrep)
     return Table(kts._1()), Table(kts._2()), \
-           Table(kts._3()), Table(kts._4())
\ No newline at end of file
+           Table(kts._3()), Table(kts._4())
+
+@handle_py4j
+@typecheck(dataset=MatrixTable,
+           pedigree=Pedigree)
+def tdt(dataset, pedigree):
+    """Performs the transmission disequilibrium test on trios.
+
+    .. include:: ../_templates/req_tvariant.rst
+
+    .. include:: ../_templates/req_biallelic.rst
+
+    Examples
+    --------
+    Compute TDT association results and export to a file:
+
+    .. testsetup::
+    
+        tdt_dataset = hc.import_vcf('data/tdt_tiny.vcf')
+
+    >>> pedigree = Pedigree.read('data/tdt_trios.fam')
+    >>> tdt_table = methods.tdt(tdt_dataset, pedigree)
+    >>> tdt_table.export('output/tdt_results.tsv')
```

Comment:
what if we `show` and put the output here rather than export? I think that's way clearer -- it reiterates the schema, field names, and that it's a table.